pkg/shell/cockpit-accounts.js

```diff
@@ -159,6 +132,84 @@ function off_account_changes(client, id) {
     $(client).off("." + id);
 }
 
+function parse_passwd_content(content) {
+    var ret = [ ];
+    var lines = content.split('\n');
+    var column;
+
+    for (var i = 0; i < lines.length - 1 /* without blank line */; i++) {
+        column = lines[i].split(':');
+        ret[i] = [];
+        ret[i]["name"] = column[0];
+        ret[i]["password"] = column[1];
+        ret[i]["uid"] = column[2];
+        ret[i]["gid"] = column[3];
+        ret[i]["gecos"] = column[4];
+        ret[i]["home"] = column[5];
+        ret[i]["shell"] = column[6];
```

Comment:
In your opinion, should we strip the line, or expect that there are no trailing characters such as ```\r``` ?